src/serial-console/azext_serialconsole/custom.py

```diff
@@ -0,0 +1,730 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import json
+import threading
+import sys
+import uuid
+import time
+import re
+import textwrap
+import numpy     # pylint: disable=unused-import
+import wsaccel   # pylint: disable=unused-import
```

Comment:
Do you use these library?

Reply:
The websocket client recommended that numpy and wsaccel be imported to improve the speed of sending data. However, this advice has changed in the last couple of weeks and now only wsaccel is recommended. I will remove the numpy import.

Reply:
Actually, upon further investigation, wsaccel is not needed if utf8 validation is disabled. I will remove it as well.